docs/getting_started.rst

```diff
@@ -34,3 +34,110 @@ If it has been a while, once can update using
+    * home/
+    * order_param/
+    * peak_integr/
+    * recp_space/
+    * ub_peak_find/
+
+        In every page-tab directory the tests are furtherd arranged in MVP-based filesystem structure:
+            * models/
+            * views/
+            * presenter/
+
+    * utility/ : It contains tests for code elements that are shared among pages and belong to the corresponding file folder (utility/).
+    * workflows/ : It contains tests with code that span among pages to test full reduction workflows.
+
+
+How to setup the garnet-data repository locally
+```````````````````
+
+Open a terminal and go to garnet repository root folder. Run:
+
+.. code-block:: sh
+
+    # add submodule
+    git submodule add https://code.ornl.gov/sns-hfir-scse/infrastructure/test-data/garnet-data tests/data/garnet-data
+
+    # initialize submodules
+    git submodule update --init --recursive
+
+To update the changes for the data-repo. Run:
+
+.. code-block:: sh
+
+    # retrieve latest changes of the repository
+    git submodule update --remote --merge
+
+
+No need to commit changes in this repository.
+More information of git-lfs can be found here https://ornl-neutrons.atlassian.net/wiki/spaces/NDPD/pages/19103745/Using+git-lfs+for+test+data .
+
+Notes.
+
+Tests that use the garnet-data repository, will need to be configured additionally for github runners.
+
+Additionally, the marker `datarepo` is used to skip tests that required garnet-repo to be present (`if not has_datarepo`), in case garnet-repo has not been configured.
+
+Instructions for CIS Testing:
+-Checkout to the PR by following the Pull-Request instructions:
+
+.. code-block:: sh
+
+    git submodule add https://code.ornl.gov/sns-hfir-scse/infrastructure/test-data/garnet-data tests/data/garnet-data
+
+To update the changes for the data-repo. Run:
+
+.. code-block:: sh
+
+     git submodule update --init --recursive
+
+No need to commit changes in this reposotiry. If a message appears to stage the garnet-data repository in this one, just run the above command.
+
+More information on git-lfs can be found here:
+
+.. code-block:: sh
+
+    https://ornl-neutrons.atlassian.net/wiki/spaces/NDPD/pages/19103745/Using+git-lfs+for+test+data
+
+
+Tests that use the garnet-data repository, will need to be configured for github runners (TBD).
+
+Additionally, the marker `datarepo` is used to skip tests that require garnet-repo to be present (`if not has_datarepo`).
+
+Instructions for CIS Testing - PRs:
+
+Checkout to the PR by following the Pull-Request instructions:
+
+.. code-block:: sh
+
+    conda activate <garnet_environment>
+    cd /path/to/my/local/garnet/repo/
+    git fetch origin pull/<PULL_REQUEST_NUMBER>/head:pr<PULL_REQUEST_NUMBER>
+    git switch pr<PULL_REQUEST_NUMBER>
+    #run tests
+    python -m pytest <item_to_test>
+    #and/or start garnet
+    garnet
+
+in case there is specific Mantid build in another conda environment, garnet can be installed there:
```

Comment:
I'm not sure what this block is supposed to mean?

Reply:
done